crowbar_framework/spec/lib/crowbar/upgrade_status_spec.rb

```diff
@@ -196,28 +196,28 @@ def new_status
       expect(subject.running?(:upgrade_prechecks)).to be false
       expect(subject.current_step).to eql :upgrade_prepare
       expect(subject.start_step(:upgrade_prechecks)).to be true
-      expect(subject.start_step(:upgrade_prechecks)).to be false
+      expect{subject.start_step(:upgrade_prechecks)}.to raise_error(Crowbar::UpgradeStatus::StartStepRunningError)
       expect(subject.running?(:upgrade_prechecks)).to be true
       expect(subject.end_step).to be true
     end
 
     it "prevents repeating steps that do not allow repetition" do
-      expect(subject.start_step(:upgrade_prepare)).to be false
-      expect(subject.start_step(:admin_backup)).to be false
-      expect(subject.start_step(:admin_upgrade)).to be false
-      expect(subject.start_step(:database)).to be false
-      expect(subject.start_step(:nodes_services)).to be false
-      expect(subject.start_step(:nodes_db_dump)).to be false
-      expect(subject.start_step(:nodes_upgrade)).to be false
+      expect{subject.start_step(:upgrade_prepare)}.to raise_error(Crowbar::UpgradeStatus::StartStepOrderError)
```

Comment:
if someone is wondering why i switched to a block syntax, it is necessary apparently when an error is raised (see http://stackoverflow.com/questions/17212754/expect-to-raise-error-and-test-fails-because-it-raises-this-error)